resources.json

```diff
@@ -264,46 +264,46 @@
 		"RAW_SOULFLOW": "Raw Soulflow",
 		"SOULFLOW": "Soulflow",
 		"ABSOLUTE_ENDER_PEARL": "Absolute Ender Pearl",
-		"ROUGH_JADE_GEM": "\u2618 Rough Jade Gemstone",
-		"FLAWED_JADE_GEM": "\u2618 Flawed Jade Gemstone",
-		"FINE_JADE_GEM": "\u2618 Fine Jade Gemstone",
-		"FLAWLESS_JADE_GEM": "\u2618 Flawless Jade Gemstone",
-		"PERFECT_JADE_GEM": "\u2618 Perfect Jade Gemstone",
-		"ROUGH_AMBER_GEM": "\u2E15 Rough Amber Gemstone",
-		"FLAWED_AMBER_GEM": "\u2E15 Flawed Amber Gemstone",
-		"FINE_AMBER_GEM": "\u2E15 Fine Amber Gemstone",
-		"FLAWLESS_AMBER_GEM": "\u2E15 Flawless Amber Gemstone",
-		"PERFECT_AMBER_GEM": "\u2E15 Perfect Amber Gemstone",
-		"ROUGH_TOPAZ_GEM": "\u2727 Rough Topaz Gemstone",
-		"FLAWED_TOPAZ_GEM": "\u2727 Flawed Topaz Gemstone",
-		"FINE_TOPAZ_GEM": "\u2727 Fine Topaz Gemstone",
-		"FLAWLESS_TOPAZ_GEM": "\u2727 Flawless Topaz Gemstone",
-		"PERFECT_TOPAZ_GEM": "\u2727 Perfect Topaz Gemstone",
-		"ROUGH_SAPPHIRE_GEM": "\u270E Rough Sapphire Gemstone",
-		"FLAWED_SAPPHIRE_GEM": "\u270E Flawed Sapphire Gemstone",
-		"FINE_SAPPHIRE_GEM": "\u270E Fine Sapphire Gemstone",
-		"FLAWLESS_SAPPHIRE_GEM": "\u270E Flawless Sapphire Gemstone",
-		"PERFECT_SAPPHIRE_GEM": "\u270E Perfect Sapphire Gemstone",
-		"ROUGH_AMETHYST_GEM": "\u2748 Rough Amethyst Gemstone",
-		"FLAWED_AMETHYST_GEM": "\u2748 Flawed Amethyst Gemstone",
-		"FINE_AMETHYST_GEM": "\u2748 Fine Amethyst Gemstone",
-		"FLAWLESS_AMETHYST_GEM": "\u2748 Flawless Amethyst Gemstone",
-		"PERFECT_AMETHYST_GEM": "\u2748 Perfect Amethyst Gemstone",
-		"ROUGH_JASPER_GEM": "\u2741 Rough Jasper Gemstone",
-		"FLAWED_JASPER_GEM": "\u2741 Flawed Jasper Gemstone",
-		"FINE_JASPER_GEM": "\u2741 Fine Jasper Gemstone",
-		"FLAWLESS_JASPER_GEM": "\u2741 Flawless Jasper Gemstone",
-		"PERFECT_JASPER_GEM": "\u2741 Perfect Jasper Gemstone",
-		"ROUGH_RUBY_GEM": "\u2764 Rough Ruby Gemstone",
-		"FLAWED_RUBY_GEM": "\u2764 Flawed Ruby Gemstone",
-		"FINE_RUBY_GEM": "\u2764 Fine Ruby Gemstone",
-		"FLAWLESS_RUBY_GEM": "\u2764 Flawless Ruby Gemstone",
-		"PERFECT_RUBY_GEM": "\u2764 Perfect Ruby Gemstone",
-		"ROUGH_OPAL_GEM": "\u2742 Rough Opal Gemstone",
-		"FLAWED_OPAL_GEM": "\u2742 Flawed Opal Gemstone",
-		"FINE_OPAL_GEM": "\u2742 Fine Opal Gemstone",
-		"FLAWLESS_OPAL_GEM": "\u2742 Flawless Opal Gemstone",
-		"PERFECT_OPAL_GEM": "\u2742 Perfect Opal Gemstone",
+		"ROUGH_JADE_GEM": "☘ Rough Jade Gemstone",
```

Comment:
These need to keep the unicode instead of being the symbols. I'm pretty sure it messes up otherwise (which is why i had them all listed out to begin with)